Client/CMakeLists.txt

```diff
@@ -14,19 +32,12 @@ add_library(chronolog_client
         src/ChronologClientImpl.cpp
 )
 
-#include/client.h
-#include/RPCClient.h
-#../ChronoAPI/ChronoLog/include/macro.h
-#../ChronoAPI/ChronoLog/include/rpc.h
-#../ChronoAPI/ChronoLog/include/RPCFactory.h
-#../ChronoAPI/ChronoLog/include/singleton.h
-#../ChronoAPI/ChronoLog/src/city.cpp
-#../ChronoAPI/ChronoLog/include/city.h)
-
+# Include directories for the library
 target_include_directories(chronolog_client PUBLIC include
         ../ChronoAPI/ChronoLog/include
```

Comment:
these test cases need to built with header files only from build/Client/include.